documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -1,5 +1,11 @@
 # Current Release notes
 
+## Version 8.11.0
+
+### New features
+
+* For Stateless and Rapid scans, the scanID and detector tool being run, are now stored in the codeLocations section of the status.json file.
```

Comment:
Hey Doug, a wee edit to squeeze in the bit about what a status.json file is and capitalize Detector.

"For Stateless and Rapid scans, the scanId and Detector tool being run, are now stored in the codeLocations section of the status.json file containing the run summary."